pkg/remediator/queue/queue.go

```diff
@@ -251,7 +295,8 @@ func (g *genericWrapper) Add(item interface{}) {
 
 // Get implements workqueue.Interface.
 func (g *genericWrapper) Get() (item interface{}, shutdown bool) {
-	return g.oq.Get()
+	obj, err := g.oq.Get(context.Background())
+	return obj, (err != nil)
```

Comment:
Could we have this return the error rather than a bool? Seems like the bool just buries information

Reply:
Nope. It has to satisfy the upstream interface, which returns a bool.
That's why it's wrapping the more expressive ObjectQueue with a wrapper.

Reply:
In this case, the context is never cancelled, so the error will always be the ErrShutdown, if not nil.